src/status_im/ui/screens/group/chat_settings/events.cljs

```diff
@@ -6,15 +6,17 @@
             [status-im.transport.message.v1.group-chat :as group-chat]
             [status-im.transport.message.core :as transport]
             [status-im.utils.handlers :as handlers]
-            [status-im.utils.handlers-macro :as handlers-macro]))
+            [status-im.utils.handlers-macro :as handlers-macro] 
+            [status-im.data-store.chats :as chats-store]
+            [status-im.data-store.messages :as messages-store]))
 
 
 ;;;; Handlers
 
 (handlers/register-handler-fx
   :show-group-chat-profile
   [re-frame/trim-v]
-  (fn [{:keys [db] :as cofx} [chat-id]] 
+  (fn [{:keys [db] :as cofx} [chat-id]]
```

Comment:
unnecessary change

Reply:
So I should **re-introduce** all those trailing whitespaces, correct ?

Reply:
@rasom I think at this point it's best to review this PR as it is, reviewing a moving target is almost more difficult than reviewing it with those changes. 
What do you think? 

To avoid this in the future we can automatically lint or prevent pushing unlinted code in a githook, @janherich seems to have the best editor setup and maybe can help. A PR that lints all the files will be necessary. 

I think the point is well taken about unnecessary changes :) We can have further discussion about this though